### DISCUSSION TITLE
CI Workflow failure - 2023 July 02

### DISCUSSION THREAD

**Discussion by scottfurry** [sqlitebrowser/sqlitebrowser#3395] [General]
After the push of fix for CODEC warning, #3393, CI workflow is showing as failure. Log output indicates the Ubuntu builds for with/without SQLCipher are aborted. From [CI log](https://github.com/sqlitebrowser/sqlitebrowser/actions/runs/5438761340/jobs/9890199533)...

```
 Warning: Aborting upload for /home/runner/work/sqlitebrowser/sqlitebrowser/appimage.tar due to failure
Error: aborting artifact upload
Total size of all the files uploaded is 33554432 bytes
File upload process has finished. Finalizing the artifact upload
Upload finished. There were 1 items that failed to upload

The raw size of all the files that were specified for upload is 36034560 bytes
The size of all the files that were uploaded is 33554432 bytes. This takes into account any gzip compression used to reduce the upload size, time and storage

Note: The size of downloaded zips can differ significantly from the reported size. For more information see: https://github.com/actions/upload-artifact#zipped-artifact-downloads 

Error: An error was encountered when uploading appimage. There were 1 items that failed to upload.
``` 

If I'm reading this correctly, there is a mis-match in upload size. The CI workflow is *calculating* that the size of upload is `36034560` but is only uploading `33554432`, a difference of 2480128 bytes (or 2.36 MiB).

I know nothing of CI workflows but I suspect that there is some file being identified as *uploadable* but possibly *MIA*.

**Comment by lucydodo** [comment]
If you look a little further up in the log you attached, you'll see the output below:
```
Uploaded /home/runner/work/sqlitebrowser/sqlitebrowser/appimage.tar (23.5%) bytes 0:8388607
Uploaded /home/runner/work/sqlitebrowser/sqlitebrowser/appimage.tar (47.0%) bytes 8388608:16777215
Total file count: 1 ---- Processed file #0 (0.0%)
Uploaded /home/runner/work/sqlitebrowser/sqlitebrowser/appimage.tar (70.6%) bytes 16777216:25165823
Uploaded /home/runner/work/sqlitebrowser/sqlitebrowser/appimage.tar (94.1%) bytes 25165824:33554431
A 503 status code has been received, will attempt to retry the upload
```
There was a **503 (Service Unavailable)** error during the upload, probably a network error in GitHub Actions.
In my experience, GitHub Actions can be a bit unreliable at times (other open source projects have similar issues).

so, it seems to be a temporary issue, I've restarted CI and everything seems to be fine now. :)
